tests/client/conftest.py

```diff
@@ -69,7 +69,7 @@ def is_using_real_services(request):
 
 @pytest.fixture(scope='session')
 def client():
-    return ScrapinghubClient(auth=TEST_ADMIN_AUTH,
+    return ScrapinghubClient(apikey=TEST_ADMIN_AUTH,
```

Comment:
> but Dash doesn't support authorisation via job_key:JWT_token, are we going to add it for consistency?

true, we already discuss this for one endpoint in Dash, we may expand it to a bigger scope, both in terms of supported endpoints and entities, e.g. we can authorize JWT tokens to operate on a project level.

but my original question was more about semantics. `apikey` scope is kinda narrow, while in reality we use other ways to authenticate clients, which are passed in the same way, so more general `auth` is more suitable in my opinion.

as for handling jwt tokens - I'd expect new client to be able to consume hex encoded jwt tokens as they are stored in environment and handle decoding and splitting it into (user, password) auth tuple internally